src/components/OcSearchBar.spec.js

```diff
@@ -0,0 +1,221 @@
+      })
+    })
+    it("should set medium search grid and search button if not enabled", () => {
+      const wrapper = getWrapper({ small: false })
+      expect(wrapper.attributes("class")).not.toContain("oc-search-small")
+      expect(wrapper.find(selectors.searchButton).attributes("size")).toBe("medium")
+    })
+  })
+  describe("icon prop", () => {
+    describe("when icon prop is not false", () => {
+      const wrapper = getWrapper({ icon: "mdi-icon" })
+      it("should add icon class to search input", () => {
+        const searchInput = wrapper.find(selectors.searchInput)
+        expect(searchInput.attributes("class")).toContain("oc-search-input-icon")
+      })
+      it("should render icon", () => {
+        const iconStub = wrapper.find("oc-icon-stub")
+        expect(iconStub.exists()).toBeTruthy()
+        expect(iconStub.attributes("name")).toBe("mdi-icon")
+      })
+    })
+    it("should not render icon if false", () => {
+      const wrapper = getWrapper({ icon: false })
+      const iconStub = wrapper.find("oc-icon-stub")
+      expect(iconStub.exists()).toBeFalsy()
+    })
+  })
+  describe("loading prop", () => {
+    describe("when loading", () => {
+      const wrapper = getWrapper({ icon: "mdi-icon", loading: true })
+      it("should show spinner", () => {
+        const spinnerStub = wrapper.find("oc-spinner-stub")
+        expect(spinnerStub.exists()).toBeTruthy()
+        expect(spinnerStub.attributes("style")).not.toBe("display: none;")
+      })
+      it("should not show icon if loading", () => {
+        const iconStub = wrapper.find("oc-icon-stub")
+        expect(iconStub.exists()).toBeTruthy()
+        expect(iconStub.attributes("style")).toBe("display: none;")
+      })
+      it("should set search input as disabled", () => {
+        const searchInput = wrapper.find(selectors.searchInput)
+        expect(searchInput.attributes("disabled")).toBe("disabled")
+      })
+      it("should set search button as disabled", () => {
+        const wrapper = getWrapper({ icon: "mdi-icon", loading: true, value: "kiran" })
+        const searchInput = wrapper.find(selectors.searchButton)
+        expect(searchInput.attributes("disabled")).toBe("true")
+      })
+    })
+    describe("when not loading", () => {
+      const wrapper = getWrapper({ icon: "mdi-icon", loading: false })
+      it("should not show spinner", () => {
+        const spinnerStub = wrapper.find("oc-spinner-stub")
+        expect(spinnerStub.exists()).toBeTruthy()
+        expect(spinnerStub.attributes("style")).toBe("display: none;")
+      })
+      it("should enable search input", () => {
+        const searchInput = wrapper.find(selectors.searchInput)
+        expect(searchInput.attributes("disabled")).toBe(undefined)
+      })
+      it("should enable search button", () => {
+        const searchInput = wrapper.find(selectors.searchInput)
+        expect(searchInput.attributes("disabled")).toBe(undefined)
```

Comment:
that looks like the same test

Reply:
ahh..should be checking for the button.
fixed